lib/p2p/NodeList.ts

```diff
@@ -87,6 +103,28 @@ class NodeList {
     return false;
   }
 
+  /**
+   * Add a reputation event to the nodes history
+   * @return true if the node was banned, false otherwise
+   */
+  public addReputationEvent = async (nodePubKey: string, type: ReputationEventType) => {
```

Comment:
If we go ahead with renaming `ReputationEventType` to `ReputationEvent`, then we can rename `type` to `event`.